### PR TITLE
[DC-3780] Accept EHR files as Parquet

### PR DESCRIPTION
**Pull request by nishanthpp93**
[DC-3780] Add parquet support

[DC-3780]: https://precisionmedicineinitiative.atlassian.net/browse/DC-3780?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: data_steward/validation/main.py [hunk 1]
```diff
@@ -274,33 +274,68 @@ def validate_submission(hpo_id: str, bucket, folder_items: list,
...
+            found_parquet_files.append(found_file)
+            found_parquet_tables.append(found_file.split('.')[0])
+        elif found_file.endswith('.jsonl'):
+            found_jsonl_files.append(found_file)
+            found_jsonl_tables.append(found_file.split('.')[0])
+        else:
+            logging.info(f"Ignoring unexpected file type: {found_file}")
+
+    # Remove any csv files that have corresponding parquet or jsonl files
+    found_csv_files = [
```
Comment (jp3477) [inline data_steward/validation/main.py:301]: So, what you're doing here is prioritizing parquet > jsonl > csv, right?

File: tests/integration_tests/data_steward/validation/main_test.py [hunk 2]
```diff
@@ -107,15 +107,17 @@ def _table_has_clustering(self, table_obj):
     @mock.patch("gcloud.gcs.LOOKUP_TABLES_DATASET_ID", dataset_id)
     def test_all_files_unparseable_output(self):
         # TODO possible bug: if no pre-existing table, results in bq table not found error
-        for cdm_table in common.SUBMISSION_FILES:
+        for cdm_table in common.SUBMISSION_CSV_FILES:
```
Comment (jp3477) [inline tests/integration_tests/data_steward/validation/main_test.py:110]: Are separate integration tests needed for jsonl and parquet files?

**Review by jp3477 — COMMENTED** [review]
@nishanthpp93 Looks good to me, I just had a couple questions first before I approve.

File: data_steward/validation/main.py (hunk 1 above)
Comment (nishanthpp93) [reply data_steward/validation/main.py:301]: Yes that's correct

File: tests/integration_tests/data_steward/validation/main_test.py (hunk 2 above)
Comment (nishanthpp93) [reply tests/integration_tests/data_steward/validation/main_test.py:110]: There's an all encompassing five_person test that has all three file types. It is a full integration test so separate ones are not needed

**Review by jp3477 — APPROVED** [review]
Thanks for working through this!

**Pull request merged**